docs/sample_config.yaml

```diff
@@ -1392,7 +1392,11 @@ sso:
     # phishing attacks from evil.site. To avoid this, include a slash after the
     # hostname: "https://my.client/".
     #
-    # By default, this list is empty.
+    # If public_baseurl is set, then the login fallback page (used by clients
+    # that don't have full support for SSO) is always included in this list.
+    #
+    # By default, this list is empty, except if public_baseurl is set (in which
+    # case the login fallback page is the only element in the list).
```

Comment:
I believe that `public_basurl` *has* to be set for SAML to work. Not sure if this changes the comment or not (it does not have to be set for CAS).

Reply:
It doesn't, this config is for both CAS and SAML so we don't expect `base_url` to be always whitelisted.